arangod/Agency/MoveShard.cpp

```diff
@@ -765,7 +780,7 @@ arangodb::Result MoveShard::abort() {
       if (_isLeader) {
```

Comment:
This branch is dead code, since as soon as there is a new leader, the abort is rejected.